src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/ListViewViewCellBinding.cs

```diff
@@ -13,39 +13,35 @@ public ListViewViewCellBinding(TestDevice testDevice) : base(testDevice)
 
 	public override string Issue => "ListView ViewCell binding";
 
-	//[Test]
-	//[FailsOnIOS]
-	//public void ListViewViewCellBindingTestsAllElementsPresent()
-	//{
-	//	App.WaitForElement(q => q.Marked("Remove"));
-	//	App.WaitForElement(q => q.Marked("Add"));
-	//	App.WaitForElement(q => q.Marked("1"));
-	//	App.WaitForElement(q => q.Marked("$100.00"));
-	//	App.WaitForElement(q => q.Marked("2"));
-	//	App.WaitForElement(q => q.Marked("$200.00"));
-	//	App.WaitForElement(q => q.Marked("3"));
-	//	App.WaitForElement(q => q.Marked("$300.00"));
-
-	//	App.Screenshot("All elements exist");
-	//}
-
-	//[Test]
-	//[FailsOnIOS]
-	//public void ListViewViewCellBindingTestsAddListItem()
-	//{
-	//	App.Tap(q => q.Button("Add"));
-	//	App.WaitForElement(q => q.Marked("4"));
-	//	App.WaitForElement(q => q.Marked("$400.00"));
-	//	App.Screenshot("List item added");
-	//}
-
-	//[Test]
-	//[FailsOnIOS]
-	//public void ListViewViewCellBindingTestsRemoveListItem()
-	//{
-	//	App.Tap(q => q.Button("Remove"));
-	//	App.WaitForNoElement(q => q.Marked("1"));
-	//	App.WaitForNoElement(q => q.Marked("$100.00"));
-	//	App.Screenshot("List item removed");
-	//}
+	[Test]
+
+	public void ListViewViewCellBindingTestsAllElementsPresent()
+	{
+		App.WaitForElement("Remove");
+		App.WaitForElement( "Add");
+		App.WaitForElement( "1");
+		App.WaitForElement("100.0");
```

Comment:
Why not using the $ symbol in this strings?

Reply:
@jsuarezruiz , In some machines, when the CurrentCulture is not set to "en-US," the string appears differently, or a box icon is displayed instead of the currency icon due to system default culture differences. To address this, I decided to use AutomationID to ensure the element's presence. If this approach is acceptable, please let me know. Alternatively, can we set the current culture at the sample level and use the dollar sign ($) string?

Reply:
I see, is fine. Keep validating the same.